### PR TITLE
Use one-sided p-values for calibration test

### PR DESCRIPTION
**Pull request by ras44**

### CODE REVIEW

File: r-package/grf/R/forest_summary.R [hunk 1]
```diff
@@ -70,6 +70,8 @@ test_calibration = function(forest) {
           "as well as the mean forest prediction as regressors, along",
           "with heteroskedasticity-robust (HC3) SEs",
           sep="\n")
+  dimnames(blp.summary)[[2]][4] <- gsub("[|]", "", dimnames(blp.summary)[[2]][4])
+  blp.summary[,4] <- blp.summary[,4]/2
```
Comment (swager) [inline r-package/grf/R/forest_summary.R]: This won't give the one-sided p-value in the case where the t-stat is negative.

Comment (ras44) [reply r-package/grf/R/forest_summary.R]: Thanks for your valuable comments.  Responding in the same order:
- re: mean.forest.prediction consistency, I believe `blp.summary[,4] <- blp.summary[,4]/2` should modify both the mean.forest.prediction and the differential.forest.prediction values.  Please let me know if I misunderstood.
- Re pretty printing, two issues:
  - "< 2e-16" decorator is dynamically created when the result is printed (verify with `res <-test_calibration(forest); res[1,4] <- 4e-16; res; res[1,4] <- 1e-16; res`
  - *-based significance decorators, these are dynamically created when the result is printed and should correspond to p-values modified after the result (verify with `res <- test_calibration(forest); res[1,4]<-0.999`)
  - In short, I think both of these are handled by the dynamic decoration that happens on print
- how about something like "... along with one-sided heteroskedasticity-robust (HC3) SEs:"?
- I've added a commit to address the negative t-stat case. Would it be helpful to have a short comment in the code here?  Something like "#convert to one-sided p-values"?

Comment (swager) [reply r-package/grf/R/forest_summary.R]: This all sounds good, thanks! The formula for going from a 2-sided p-value to a 1-sided one is `ifelse(t < 0, 1 - pval / 2, pval / 2)`. Also, nitpick: we don't need to use the `sign` command on the t-stat since we're only checking if it's smaller than 0.

Comment (ras44) [reply r-package/grf/R/forest_summary.R]: agreed!  the `sign()` was an unfortunate relic of a first attempt to do the calculation without an ifelse.  Also, previously it might have looked like I was using the formula like ifelse(..., 1-pval, pval), but that was because I was setting pval to pval/2 in the prior line in the code.  It should now be a little more clear, all in one line.  I've also added the update to the function description and a comment.

**Comment by swager** [comment]
In response to #396

**Comment by ras44** [comment]
Great!  Thanks for your helpful reviews, @swager
